packages/pipeline-editor/src/index.ts

```diff
@@ -149,6 +155,16 @@ const extension: JupyterFrontEndPlugin<void> = {
       command: commandIDs.submitNotebook,
       rank: -0.5
     });
+
+    const updateTheme = (widget: any): void => {
+      const pipelineEditor = widget.content;
+      pipelineEditor.themeChanged(themeManager.isLight(themeManager.theme));
```

Comment:
since this function has been simplified should this just be one line or do you think keeping it separate is cleaner?

Reply:
good catch. it is changed to single line